tests/ttnn/unit_tests/operations/eltwise/test_composite.py

```diff
@@ -235,6 +239,25 @@ def test_unary_composite_hardsigmoid_ttnn(input_shapes, device):
     assert comp_pass
 
 
+@pytest.mark.parametrize(
+    "input_shapes",
+    (
+        (torch.Size([1, 1, 32, 32])),
+        (torch.Size([1, 1, 320, 384])),
+        (torch.Size([1, 3, 320, 384])),
+    ),
+)
+def test_unary_composite_hardsigmoid_bf8b(input_shapes, device):
+    in_data1, input_tensor1 = data_gen_with_range_dtype(input_shapes, -100, 100, device, False, False, ttnn.bfloat8_b)
```

Comment:
I am curious, would we generate a test for many different datatypes, making the last parameter `ttnn.bfloat8_b` a test param?

Reply:
yes. by default `data_gen_with_range` uses dtype=ttnn.bfloat16, we use `data_gen_with_range_dtype` when there needs to be specific dtype tested (usually for a GH issue)